test/wasmBenchmarker/benchmark.py

```diff
@@ -45,14 +46,15 @@
     "red-black": 4000000,
     "rsa": 0,
     "salesman": 840,
-    "simdMandelbrot": 775007,
+    "simdMandelbrotFloat": 775007,
+    "simdMandelbrotDouble": 775007,
     "simdNbody": -0.1691057,
     "simd_matrix_multiply": 3920.0,
 }
 # https://benchmarksgame-team.pages.debian.net/benchmarksgame/description/simple.html#simple
-gameTests = ["mandelbrot", "nbody", "gregory", "fannkuch", "k_nucleotide"]
+gameTests = ["mandelbrotFloat", "nbody", "gregory", "fannkuch", "k_nucleotide"]
```

Comment:
`mandelbrotDouble` is omitted.
Is this intended?

Reply:
Yes, I wanted to add only the float version.
Is it okay this way, or should I add the double version, too?

Reply:
I see.